src/main/java/com/codegym/jrugotom5/entity/Advert.java

```diff
@@ -1,4 +1,36 @@
 package com.codegym.jrugotom5.entity;
 
+import java.util.Date;
+
 public class Advert {
+    private Long id;
+    private String name;
+    private String description;
+    private User createdBy;
+    private Date createdDate;
+    private Date endDate;
+    private Boolean isActive;
+
```

Comment:
My task is to create an Advert entity, so please, delete this entity and wait for my pull request to be approved. After that, you will be able to merge this entity from the main to your branch

Reply:
I haven't implemented Advert entity, just created some placeholders with fields and methods required for my task. Anyway I will need to update my code after you implement the Advert.

Reply:
Done.